commands/networkscommands/commands.go

```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "network",
-			Usage:       "Used for Cloud Networks network operations",
+			Usage:       "Set up new networks to use by servers.",
```

Comment:
`Set up new networks` is a bit of a tongue twister.

Thinking outloud. `Software defined networks used by Cloud Servers`